plaidml/bridge/openvino/ops/deformable_convolution.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/lib/ops.h"
+#include "plaidml/op/op.h"
+
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+using namespace plaidml::edsl;
+using namespace plaidml::op::lib;
+
+namespace {
+std::vector<int> cast_vector(std::vector<int64_t> vec) {
```

Comment:
Please check if this function can be removed if you change the broadcast op to accept int64_t as arguments.

Reply:
ok, i will change the broadcast op to accept int64_t as arguments.